game/libxml++-impl.hh

```diff
@@ -19,7 +19,7 @@ namespace xmlpp {
 	static inline void set_first_child_text(Element* element, const Glib::ustring& content) {
 		return element->set_child_text(content);
 	}
-#elif LIBXMLPP_VERSION_3_0
+#elif defined(LIBXMLPP_VERSION_3_0)
```

Comment:
And here.

The problem with changing `#if`to `#if defined()``

Check this example to understand why: https://onlinegdb.com/BJj90o7OU

Reply:
but wait just a minute, if LIBXMLPP_VERSION_3_0 is just a symbol without literals (defined by `#define LIBXMLPP_VERSION_3_0` instead of something like `#define LIBXMLPP_VERSION_3_0 1`, the correct form is with *defined* operator!

probably the source of troubles is different define forms on different platforms. MSVC don't understand the code without *defined* and it do it right.

Reply:
But the module defines it to be 0.

Reply:
Here: https://github.com/performous/performous/blob/39dad10cc289d2eb6e73cba300ae0602fd718048/cmake/Modules/FindLibXML%2B%2B.cmake#L19-L20

A possible solution would be to explicitly use `#if LIBXMLPP_VERSION_2_6 == 1`